.ci/static-checks.sh

```diff
@@ -299,6 +299,27 @@ static_check_go_arch_specific()
 	eval "$linter" "${linter_args}" "$dirs"
 }
 
+static_check_rust_arch_specific()
+{
+	local -r libs=("rustjail" "oci")
```

Comment:
I know this is only for rust agent but I wonder if would be good to make this a ENV variable that the script checks and in an repo we are free to set  what libs we want to analyse  or not.

Reply:
fixed